templates/new_request_page.hbs

```diff
@@ -47,6 +46,5 @@
       requestPath: {{json (page_path 'request' id=answer_bot.request_id)}}
     },
   };
-
   renderNewRequestForm(settings, props, container);
-</script>
+</script>
```

Comment:
There are still some changes in the formatting here (some removed empty lines)

Reply:
Removed these changes